readme.md

```diff
@@ -46,7 +61,7 @@ Number of columns to wrap the text to.
 
 - [slice-ansi](https://github.com/chalk/slice-ansi) - Slice a string with ANSI escape codes
 - [chalk](https://github.com/chalk/chalk) - Terminal string styling done right
-
+- [jsesc](https://github.com/mathiasbynens/jsesc) - Generate ascii only output from unicode strings. This is useful for creating testing fixtures.
```

Comment:
+1 for keeping it, seems very useful. Developers like me probably wouldn't know about it otherwise.

Could also add a link for `strip-ansi`.